src/Raven.Server/ServerWide/ClusterStateMachine.License.cs

```diff
@@ -0,0 +1,207 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Raven.Client.Exceptions.Commercial;
+using Raven.Client.ServerWide;
+using Raven.Server.Commercial;
+using Raven.Server.Json;
+using Raven.Server.ServerWide.Commands;
+using Raven.Server.ServerWide.Commands.ETL;
+using Raven.Server.ServerWide.Commands.Indexes;
+using Raven.Server.ServerWide.Commands.PeriodicBackup;
+using Raven.Server.ServerWide.Context;
+using Voron.Data.Tables;
+
+namespace Raven.Server.ServerWide;
+
+public sealed partial class ClusterStateMachine
+{
+    private const int MinBuildVersion54116 = 54_116;
```

Comment:
Why is it special build number?

Reply:
We want it to happen starting from the latest build.
A mixed cluster with `5.4.110` and `5.4.118` may result in inconsistent state (an index that was deployed on the `5.4.110` node and not on the `5.4.118` node for example).
@efratshenhar this needs to be changed to `54_118`.

Reply:
OK

Reply:
Done